content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -98,7 +86,7 @@ which are nested under the parent Git commit.
 
 <admon>
 
-The live experiments row for an experiment is displayed only if its parent Git
+The live experiment row for an experiment is displayed only if its parent Git
```

Comment:
@mvshmakov We can remove this statement now, because of detached experiments, right?

Reply:
Yes, we can. But probably we don't want to remove it, we want to add a paragraph about the detached experiments instead, WDYT?

Reply:
@mvshmakov yes, we should write about detached exps. It may also need a separate sub-section of its own. Can you pls share a rough draft of what to write (bullet points will be fine - I'll revise and include it).

Reply:
* Detached experiments are the experiments, for those we cannot find a baseline commit in the remote repository. It can be: (1) local non-pushed commit; (2) commit that got removed by some mutative git action (e.g., rebase, hard reset with a push, squash commit, etc.).
* It is not a desired way to iterate on the experiments without having them assigned to a baseline commit.
* To "fix" them, one have to either push the missing commit to the git remote, or to remove the experiment from the UI and iterate on it once again with the correct baseline commit.
* To check which commit is a parent one, the user can do `git log --first-parent {sha1}`, where `sha1` is the copied exp SHA. I'm not sure if we should add it tbh, it must be a part of the UI - I'll send a PR.

Reply:
> * To check which commit is a parent one, the user can do `git log --first-parent {sha1}`, where `sha1` is the copied exp SHA. I'm not sure if we should add it tbh, it must be a part of the UI - [I'll send a PR](https://github.com/iterative/studio/issues/6485).

I think it might be overkill to worry about what commit is the parent one, and it's not usual to push a specific commit. In most cases, suggesting that the user who generated it do a `git push` is probably enough.

Reply:
Thanks @mvshmakov @dberenbaum.
I've added a section for detached experiments [here](https://github.com/iterative/dvc.org/pull/4638/commits/ede1f93a174f68d79eddfc4e3e6c9171940d1b76).